src/gui_common/ChemicalEquation.cs

```diff
@@ -143,7 +143,7 @@ public bool ShowFullSecondText
     /// <summary>
     ///   If true this will automatically check the set process for changes
     /// </summary>
-    public bool AutoRefreshProcess { get; set; } = true;
+    public static bool AutoRefreshProcess { get; set; } = false;
```

Comment:
Why is this static? This means that *all* chemical equations in the game share this, including the process panel.